alexandria/core/models.py

```diff
@@ -172,6 +172,22 @@ class File(UUIDModel):
         Document, on_delete=models.CASCADE, related_name="files"
     )
 
+    UNDEFINED = "undefined"
+    COMPLETED = "completed"
+    ERROR = "error"
+
+    STATUS_CHOICES = (
+        UNDEFINED,
+        COMPLETED,
+        ERROR,
+    )
+    UPLOAD_STATUS_TUPLE = (
+        (status_choice, status_choice) for status_choice in STATUS_CHOICES
+    )
```

Comment:
This will create a generator and won't be usable twice. Better:

```suggestion
    UPLOAD_STATUS_TUPLE = tuple([
        (status_choice, status_choice) for status_choice in STATUS_CHOICES
    ])
```